docs/evaluation/how_to_guides/datasets/manage_datasets_in_application.mdx

```diff
@@ -13,70 +13,96 @@ Before diving into this content, it might be helpful to read the following:
 
-![](./static/new_dataset.png)
+## Set up your dataset 
 
-Then, enter the relevant dataset details, including a name, optional description, and dataset type. Please see the [concepts](../../concepts#datasets-and-examples) for more information on dataset types. For most flexibility, the key-value dataset type is recommended.
+To create a dataset, head to the Datasets & Experiments page in LangSmith, and click `+ Dataset`. You'll have 3 options for
+getting started:
 
-![](./static/enter_dataset_details.png)
+TODO: INSERT IMAGE WHEN COPY FINAL
 
-You can then add examples to the dataset by clicking on "Add Example". Here, you can enter the input and output as JSON objects.
+* **Create from CSV:** Drag and drop your CSV into LangSmith and your data will be imported automatically. Each row will correspond to a new dataset example.
+* **Create from scratch:** If you prefer to upload data from the SDK or manually, you can instead set up your dataset before seeding any data. 
 
-![](./static/add_manual_example.png)
+   LangSmith datasets store arbitrary JSON objects. We recommend (but do not require) that you define a schema for your dataset.
+   Dataset schemas are defined with standard [JSON schema](https://json-schema.org/), with the addition of a few [custom types](/reference/data_formats/dataset_json_types) 
+   that make it easier to define commonly used LLM data primitives like messages and tool definitions.
 
-## Dataset schema validation
+   As part of your schema definition, certain fields allow you to click `+ Transformations` which enable common
+   preprocessing on your data to occur when you add examples to your dataset, such as converting LangSmith traces 
+   into standard openAI message formats, or dropping fields that aren't defined in your schema. For the full list 
+   of available transformations, see [our reference](/reference/data_formats/dataset_transformations)
 
-If you are creating a key-value dataset, you may optionally define a schema for your dataset. All examples you create will be validated against this schema.
+* **Create from template**: If you plan to collect production traces in your dataset from LangChain 
+[ChatModels](https://python.langchain.com/api_reference/core/language_models/langchain_core.language_models.chat_models.BaseChatModel.html) 
+or from OpenAI calls using the [LangSmith OpenAI wrapper](/observability/how_to_guides/tracing/annotate_code#wrap-the-openai-client), we offer a prebuilt schema and transformation that makes it easy to
+collect these runs in format that is easy to use downstream. This is entirely customizable, and does not lock your dataset into a set format. 
 
-![Defining dataset schema](./static/dataset_schema_definition.png)
+   Please see the [dataset transformations reference](/reference/data_formats/dataset_transformations) for more information. 
```

Comment:
would be great (now or as follow up) to add transformations to concepts and link to that instead